snowflake/cmd/policyRecommendation.go

```diff
@@ -0,0 +1,285 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+
+	"antrea.io/theia/snowflake/pkg/infra"
+	"antrea.io/theia/snowflake/pkg/udfs"
+	"antrea.io/theia/snowflake/pkg/utils/timestamps"
+)
+
+const (
+	staticPolicyRecommendationFunctionName = "static_policy_recommendation"
+	preprocessingFunctionName              = "preprocessing"
+	policyRecommendationFunctionName       = "policy_recommendation"
+	defaultFunctionVersion                 = "v0.1.0"
+	defaultWaitTimeout                     = "10m"
+	// Limit the number of rows per partition to avoid hitting the 5 minutes end_partition() timeout.
+	partitionSizeLimit = 30000
```

Comment:
Could we use 50k now (latest Snowflake version)?

Reply:
Yes